### ISSUE TITLE
Performance decline when using pretrained embedding.

### ISSUE THREAD

**Issue by fooSynaptic** [baidu/DuReader#40]
I use the pre-trained embedding (fasttext zh-300-vec) rather than the random initialized embedding.
The loss was lower in evaluate. But the performance declined when inference.
Is it the OOV issue or other problem.

**Comment by lkliukai** [comment]
We have investigated some pre-trained embeddings as untrainable input of the encoder, none of each showed significance outperformance. But trainable embeddings (e.g. ELMO, BERT) could benefit the model significantly.
And YES, it may be the OOV issue, but it is not a final conclusion.

**Comment by fooSynaptic** [comment]
thanks
